hack/jenkins/benchmark-dockerized.sh

```diff
@@ -21,7 +21,7 @@ set -o xtrace
 
 retry() {
   for i in {1..5}; do
-    "$@" && return 0 || sleep $i
+    "$@" && return 0 || sleep ${i}
```

Comment:
quoting is at least as important as the braces here.

Technically in this instance $i "$i" ${i} "${i}" all do the same here. I think we should either do $i (simplest) or else "${i}" (safest)

Reply:
[per the Google shell style guide](https://google.github.io/styleguide/shell.xml?showone=Variable_expansion#Variable_expansion) (which we roughly follow), `"${i}"` would be preferred.

Reply:
Resolved, thanks